ui/components/ui/box/index.js

```diff
@@ -1,6 +1 @@
-export {
-  default,
-  MultipleSizes,
-  ValidBackgroundColors,
-  ValidBorderColors,
-} from './box';
+export { default, MultipleSizes, MultipleSizesAndAuto } from './box';
```

Comment:
`ValidBackgroundColors` `ValidBorderColors` weren't being used anywhere and I can't see them being needed anywhere yet. Also adding `MultipleSizesAndAuto` to be used in the `Typography` component